folium/utilities.py

```diff
@@ -44,14 +39,14 @@ def _validate_coordinates(coordinates):
     if _isnan(coordinates):
         raise ValueError('Location values cannot contain NaNs, '
                          'got:\n{!r}'.format(coordinates))
-    coordinates = _locations_tolist(coordinates)
+    coordinates = _iter_tolist(coordinates)
     return coordinates
 
 
-def _locations_tolist(x):
+def _iter_tolist(x):
```

Comment:
@Conengmo sorry for the amount of noise in this PR. The fix for #928 is actually only this part :grimacing: and its use in the heatmap plugin.

Reply:
no problem!